### PR TITLE
WIP: Added cp file RPC method

### PR DESCRIPTION
**Pull request by m3dwards**
Untested and a draft for now

### CODE REVIEW

File: src/warnet/server.py [hunk 1]
```diff
@@ -152,6 +152,19 @@ def tank_debug_log(self, network: str, node: int) -> str:
             self.logger.error(msg)
             raise ServerError(message=msg) from e
 
+    def tank_cp_file(self, network: str, node: int, src) -> str:
+        """
+        Copy file from <node>
+        """
+        wn = Warnet.from_network(network, self.backend)
+        try:
+            result = wn.container_interface.get_bitcoin_debug_log(wn.tanks[node].container_name)
```
Comment (willcl-ark) [inline src/warnet/server.py:161]: should this really be `get_bitcon_debug_log`?

Comment (m3dwards) [reply src/warnet/server.py:161]: perhaps not..

**Comment by josibake** [comment]
Can you add a motivation to the description? IIRC, at one point we discussed copying files *to* the RPC server, but I can't remember why we needed to copy from containers

**Comment by m3dwards** [comment]
Copying files was required for tests to pass and I just wanted a handy way to test copying files manually so started adding it to the CLI. I had a random stash of working towards that so just pushed it up to not lose it.

**Comment by willcl-ark** [comment]
I think this should work much more nicely-er with the fixes in #277 (to `get_file()`)

**Comment by josibake** [comment]
Circling back to this, with helm and the new commanders approach, do we still need this?

**Comment by willcl-ark** [comment]
I think this can prob be closed. If anything we'll add another wrapper + picker directly around kubectl

**Pull request closed without merging**